inexor/engine/main.cpp

```diff
@@ -1198,10 +1199,18 @@ ICOMMAND(cef_reload, "", (),
 ICOMMAND(cef_focus, "b", (bool *b),
     if (cef_app.get()) cef_app->setFocus(*b); );
 
+#include <csignal>
```

Comment:
its for testing the crashreporter and will be removed as last step before a merge